### PR TITLE
Add extracted text to MongoDB

### PR DESCRIPTION
**Pull request by AlexF301**
Modified DataStorage class to include text extraction. The table extracted_text gets added to during the extraction process. Currently providing the filename and text to Mongo.

This will help to determine how many attachments the system has extracted text from. Furthermore, will be of use for upcoming changes to Client dashboard to display the extraction progress.

### CODE REVIEW

**Comment by AlexF301** [comment]
Ready for Review

File: mirrulations-core/src/mirrcore/data_storage.py [hunk 1]
```diff
@@ -8,6 +8,7 @@ def __init__(self):
         self.documents = database['documents']
         self.comments = database['comments']
         self.attachments = database['attachments']
+        self.extraction = database['extracted_text']
```
Comment (jack11wagner) [inline mirrulations-core/src/mirrcore/data_storage.py:11]: To align with our disk storage we may want the database to be named `comments_extracted_text`

Comment (jack11wagner) [reply mirrulations-core/src/mirrcore/data_storage.py:11]: Since we may eventually be adding tables for `documents_extracted_text`

**Comment by AlexF301** [comment]
Do Not Merge

**Comment by AlexF301** [comment]
MongoDB set for removal. Closing this pull request

**Pull request closed without merging**